app/generator/content/schedule/part3/get-table.js

```diff
@@ -10,27 +12,29 @@ const getTable = (schedule) => {
     fillColor: '#d9d9d9',
     table: {
       headerRows: 1,
-      widths: ['*', ...schedule.map(x => ('*'))],
+      widths: ['*', '*', '*'],
       body: [
         [
           { text: 'Payment type', style: 'tableHeader' },
-          ...schedule.map(x => ({ text: x.paymentType, style: 'tableHeader' }))
-        ],
-        [
-          { text: 'Amount', style: 'tableHeader' },
-          ...schedule.map(x => ({ text: toCurrencyString(x.value), style: 'tableNumber' }))
-        ],
-        [
           { text: 'Payment period', style: 'tableHeader' },
-          ...schedule.map(x => ({ text: x.period, style: 'tableNumber' }))
-        ]
+          { text: 'Amount', style: 'tableHeader' }
+        ],
+        ...schedule.map(x => ([
+          { rowSpan: getRowSpan(x.paymentType, rowsCount, x.order), text: x.paymentType, style: 'tableNumber' },
+          { text: x.period, style: 'tableNumber' },
```

Comment:
Payment periods are different to template.

`March 2023` to your `Mar 2023`
`February 2023 to April 2023` to your `Feb-Apr 2023`

Need to verify this is expected.

Reply:
Sample of generated schedule was presented to the team on 12/06/2023, everyone agreed that it should be presented as it is for now. If further consultation requires change to be made, new ticket would be raised to that effect.